src/components/Dialog.tsx

```diff
@@ -116,7 +116,7 @@ export const Modal = styled.div<{ color: Color }>`
   padding: 0.5em;
   position: absolute;
   top: 0;
-  width: 100%;
+  width: calc(100% - 1em);
```

Comment:
o nice, i have a different fix in my PR that does `right: 0` which doesnt need a custom calculation

Reply:
fixed now so this can be removed :3